test/integration/configserver/default_spec.rb

```diff
@@ -0,0 +1,7 @@
+# mongodb | mongod
+
+describe service('mongodb') do
```

Comment:
service name is not always `mongodb`
The BATS side allows `mongodb` or `mongod`